packages/types/src/base.ts

```diff
@@ -125,14 +125,14 @@ export type QuoteResponse = {
   sources?: Array<SwapSource>
 }
 
-export type Quote<C extends ChainTypes> = {
+export type Quote<SellAssetType extends ChainTypes> = {
```

Comment:
standard generic convention makes use of single character uppercase names ie. `T` for type. this is usually sufficient for clarity especially when dealing with a single generic parameter. we may want a quick discussion on if we think this is reasonable, or if we would like to make use of more verbose and descriptive generic naming conventions. not a blocker, just something of note if anyone has any strong opinions one way or the other.

Reply:
I changed it back. going convention is fine with me